main.py

```diff
@@ -0,0 +1,54 @@
+from pypanther import PantherLogType, PantherSeverity, get_panther_rules, register
+from pypanther.rules.aws_cloudtrail_rules.aws_cloudtrail_account_discovery import AWSCloudTrailAccountDiscovery
+from pypanther.rules.aws_cloudtrail_rules.aws_cloudtrail_stopped import AWSCloudTrailStopped
+from pypanther.rules.aws_cloudtrail_rules.aws_codebuild_made_public import AWSCloudTrailCodebuildProjectMadePublic
+from pypanther.rules.aws_cloudtrail_rules.aws_console_login_without_mfa import AWSConsoleLoginWithoutMFA
+from pypanther.rules.aws_cloudtrail_rules.aws_console_login_without_saml import AWSConsoleLoginWithoutSAML
+from pypanther.wrap import exclude, include
+
+from rules.aws_cloudtrail_rules.my_first_custom_rule import MyFirstCustomRule
+from rules.aws_cloudtrail_rules.my_first_tuned_rule import MyFirstTunedRule
+
+########################################################
+## Variables
+
+onboarded_log_types = [
```

Comment:
I like this of defining the onboarded log types and then onboarding those rules! That's an amazing way of customers going 0-1 with pypanther.

I wonder if an iteration of this could be hitting the API to get this actual list 🤔